### ISSUE TITLE
Lint/Void false positive result

### ISSUE THREAD

**Issue by kvokka** [rubocop/rubocop#9792]
## Expected behavior

```ruby
class Foo
  def foo
    1..100.times do |_n|
      break unless rand > 0.5
    end
    raise
  end
end
```

be ok

## Actual behavior

```bash
$ be rubocop
foo.rb:5:5: W: Lint/Void: Literal 1..100.times do |_n|
      break unless rand > 0.5
    end used in void context.
    1..100.times do |_n| ...
    ^^^^^^^^^^^^^^^^^^^^
```

If replace to `(1..100).times` then everything works as it should

## RuboCop version

```
1.14.0 (using Parser 3.0.1.1, rubocop-ast 1.5.0, running on ruby 2.7.2 x86_64-darwin19)
```

**Comment by koic** [comment]
I couldn't reproduce it.

> foo.rb:9:5: W: Lint/Void: Literal 1..50.times do

Can you provide me the whole foo.rb? The code snippet may be missing.

**Comment by kvokka** [comment]
> Can you provide me the whole foo.rb? The code snippet may be missing.

@koic 
Sorry, my bad, cut out too much. Fixed the first message with the correct snippet
